src/components/ScoreBoard/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Container from "../Container";
+import styles from "./index.module.css";
+
+type ScoreType = {
+  className: string;
```

Comment:
Make className optional.
```suggestion
  className?: string;
```